axios migration refactoring useFetchClient and getFetchclient (#15400)
strapi/strapi: docs/docs/core/hooks/use-fetch-client.mdx

### Base URL

> The default URL will be the one defined in the _getFetchClient_ utility

Reviewer: ```suggestion
The default URL is the one defined in the _getFetchClient_ utility
```
